47.md

```diff
@@ -63,6 +63,267 @@ Response:
 }
 ```
 
+Below are the commands that are defined in this NIP.
+
+### Pay Invoice
+
+Request:
+```jsonc
+{
+    "method": "pay_invoice",
+    "params": {
+        "invoice": "lnbc50n1..." // invoice to pay, required
+    }
+}
+```
+
+Response:
+```jsonc
+{
+    "result_type": "pay_invoice",
+    "error": {
+        "code": "UNAUTHORIZED",
+        "message": "human readable error message"
+    },
+    "result": {
+        "preimage": "0123456789abcdef..." // preimage of the payment
+    }
+}
+```
+
+### Keysend
+
+Request:
+```jsonc
+{
+    "method": "pay_keysend",
+    "params": {
+        "amount": 123, // invoice amount in msats, required
+        "pubkey": "03...", // payee pubkey, required
+        "message": "some message", // keysend message, optional
```

Comment:
this will actually also be part of a tlv record, isn't it? 
do we need this as a special key here?